CHANGELOG.rst

```diff
@@ -14,6 +14,15 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[0.14.0] - 2017-01-15
```

Comment:
We've got another `0.14.0` in the works [here](https://github.com/edx/edx-enterprise/pull/30/files#diff-de0a9b67ffe2874a076b5fdb15bad484R17), just FYI to you and @saleem-latif